core/backend/src/IModelHost.ts

```diff
@@ -196,6 +196,10 @@ export interface IModelHostOptions {
    */
   allowSharedChannel?: boolean;
 
+  /**
+   * Use native instance functions for element, model, and aspect CRUD operations.
```

Comment:
As an app developer, when and why should I set this to true? What will happen if I do?